apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracerBuilder.java

```diff
@@ -60,10 +60,10 @@
 import java.util.concurrent.TimeUnit;
 
 public class ElasticApmTracerBuilder {
-
     /**
      * See {@link co.elastic.apm.attach.ElasticApmAttacher#TEMP_PROPERTIES_FILE_KEY}
      */
+    @SuppressWarnings("JavadocReference")
```

Comment:
removes the javadoc error/warning from the comment just above due to field visibility